stable/admin/values.yaml

```diff
@@ -228,6 +228,29 @@ admin:
   podAnnotations: {}
     # vault.hashicorp.com/agent-inject: true
 
+  ## Transparent Data Encryption (TDE) secrets
+  # Storage passwords can be managed and stored as Kubernetes
+  # secrets. TDE secrets for all databases should be created and 
+  # provided before deploying NuoDB. For example:
+  #   kubectl create secret generic demo-tde-secret -n nuodb \
+  #     --from-literal target='topSecret'
+  #
+  # Note: TDE should be enabled on database layer using
+  #   SQL> alter database change encryption type AES128;
+  #
+  # Storage password rotation is performed by updating te secret and 
+  # providing current, new target and historical (optional) passwords. 
```

Comment:
This needs updating since we don't require the current password anymore.

Reply:
Good catch! It looks like I missed this one after removing the current from the service.